docs/src/main/paradox/compatibility-guidelines.md

```diff
@@ -2,7 +2,7 @@
 
 ## Binary Compatibility Rules
 
-Apache Pekko HTTP follows the same binary compatibility rules as Akka itself.
+Apache Pekko HTTP follows the same binary compatibility rules as Pekko itself.
```

Comment:
Are there any guidelines about when to use "Apache" with the name? In the next change below "Apache" is used...

Reply:
I'll review the changes and update the uses of Pekko by itself.

We should use 'Apache Pekko' but not exhaustively. We should always start with 'Apache Pekko' but I think it is ok to just use Pekko later in the article.

In the example you highlight above, we need to match the 2 references. Both should probably be 'Apache Pekko' but if we've mentioned 'Apache Pekko' before in the article, there could be instances where it reads better to just use Pekko.